### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by Dilatorily**
[Review on Reviewable](https://reviewable.io/reviews/dudang/golt/1)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/dudang/golt/pull/1" target="_blank">ReviewNinja

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 dudang
+Copyright (c) 2015 dudan test
```
Comment (Dilatorily) [inline LICENSE:3]: Should be dudang

Comment (PhilibertDugas) [inline LICENSE:3]: **dudang** was fine

Comment (Dilatorily) [inline LICENSE:3]: !fix

**Pull request closed without merging**